tests/acceptance/features/coreApiTrashbin/trashbinSharingToShares.feature

```diff
@@ -27,6 +27,7 @@ Feature: using trashbin together with sharing
     And as "Brian" the folder with original path "/Shares/renamed_shared" should not exist in the trashbin
     Examples:
       | dav-path-version |
+      | old              |
```

Comment:
we do not have old trashbin path

Reply:
These scenarios uses following when step:
```feature
When user "Brian" deletes folder "/Shares/renamed_shared" using the WedDAV API
```
So, using all dav paths.